### ISSUE TITLE
Unable to disable recent chats

### ISSUE THREAD

**Issue by matrixx740** [redsolution/xabber-android#790]
In version 2.2(454), I'm unable to disable "Recent chats" from roster.

**Comment by andrewnenakhov** [comment]
You can disable recent chats by quitting application.

Seriously, it's new main focus of the app and it can't and won't be turned off.

**Comment by Scapponian** [comment]
Not being able hide recent chats is very annoying. I'll be switching to a new xmpp app.

**Comment by andrewnenakhov** [comment]
@Scapponian we can't please everybody, so, good luck to you. 

We intend to make a continuous chat experience across platforms (see how Xabber for Web works), and obsolete concept of making interface around roster is standing in our way. It has to leave to make room for great things we plan to do in the future.

**Comment by matrixx740** [comment]
I understand you are experimenting with the UI, but wouldn't it make more sence include tiny bit of options so users can actually set what they prefer? Moreover the option existed in previous versions, so you had to do more work to remove the option than to keep it.
Unfortunately I can't start fork of every single program I use when they change nonsence like this, so filling bugreports and feature requests is the only thing i can do to change thing to be usable (again).

**Comment by andrewnenakhov** [comment]
@matrixx740 every tiny bit of options needs to be supported. Xabber already has way too many options that make it very hard to test. ANd as for UI, you see, users are very accustomed to seing their contact list in XMPP, but it's not the way to move forward. When redesigning chat app we were closer to removing roster, not 'recent chats', because clearly trend for massively adopted apps is building interface around recent chats. That works really nice in desktop (browser) version, too. 

So if we have a choice of building an app that masses will understand, or catering to needs of few orthodox roster-centrists, we have to choose the former, because we want XMPP protocol to thrive and unite the instant messaging world, which is currently divided between few walled gardens of FB messenger, Whatsapp, Telegram, etc. So If someone does not want to have a modern chat app based on XMPP and _really_ needs and old-school xmpp client, well, there are many other clients for Android. Sorry.

**Comment by matrixx740** [comment]
OK, though I still don't see a reason why I have to have all contacts displayed twice, just with diferent sorting (last chat time vs. alphabet)

**Comment by andrewnenakhov** [comment]
@matrixx740 well, actually default is to display contact with status (and soon, 'last seen' info if contact is offline). That option to display last message in roster was made because one guy literally BEGGED me  for several days to have this option so I caved in under such pressure. )

**Comment by matrixx740** [comment]
No, let me rephrase that. the contact screen looks like Recent chats: Bob, Alice, David, Carol. Myself: Alice, Bob, Carol, David. I literally see my whole roster twice, just sorted differently. What is benefit of that?
I've just noticed I can sort the second part - the actual roster - by alphabet or status, why not add "last chat time" and keep it just once?
